### PR TITLE
Added logic to hide portion of schema entity creator when not clickable

### PR DESCRIPTION
**Pull request by Invocatis**

### CODE REVIEW

File: src/hyperfiddle/ide/fiddles/schema_attribute.cljs [hunk 1]
```diff
@@ -98,30 +101,32 @@
...
-         (doall
-           (for [[k _] (hf/spread-attributes ctx)
-                 :when (not= "db" (namespace k))]
-             (field [k] ctx)))
-
-         ]))))
+         [:div {:style {:display (if valid-attr? "block" "none")}}
+           (field [:db/doc] ctx nil {:disabled (not valid-attr?)})
+           (field [:db/unique] ctx hyperfiddle.ui/hyper-control {:disabled (not valid-attr?)
+                                                                 :options "unique-options"})
```
Comment (dustingetz) [inline src/hyperfiddle/ide/fiddles/schema_attribute.cljs]: While you are here, I believe there is an :option-label prop to apply a function to the record to generate clean labels for the picklist values.

Comment (Invocatis) [reply src/hyperfiddle/ide/fiddles/schema_attribute.cljs]: There's a :label-fn that changes the title area of the component. I didn't find anything called :option-label. 

I could try to implement this if it's a feature we want.

Comment (dustingetz) [reply src/hyperfiddle/ide/fiddles/schema_attribute.cljs]: https://github.com/hyperfiddle/hyperfiddle/blob/0a1d164f6e02ebf146395fff24b7c583782b92af/src/hyperfiddle/ui/select%24.cljs#L115

Usage here, :district/region renderer defined in fiddle/cljs 

http://tank.hyperfiddle.net/:demo.seattle!neighborhood/~entity('$',(:neighborhood!name,'Georgetown'))

Comment (dustingetz) [reply src/hyperfiddle/ide/fiddles/schema_attribute.cljs]: There may be a reason that it is not used on this page - it used to be used on this page

Comment (Invocatis) [reply src/hyperfiddle/ide/fiddles/schema_attribute.cljs]: Oh word I see it. I'll give it a go

**Pull request merged**